### ISSUE TITLE
Private Functions are not private

### ISSUE THREAD

**Issue by RandomNoun7** [joeypiccola/PSPuppetOrchestrator#10]
Functions added to the `Private` folder of the module source code, after compilation into the PSM1 file are then all exported as public functions anway. 

The `build` PSake tasks seems to need to be modified to ensure that there is an `Export-ModuleMember` command added that only exports the public function names.

**Comment by joeypiccola** [comment]
Yup... the `psake` tasks are a bit of a mess 👎 . Just started to look at those, hoping to revert to 💯 `PowerShellBuild` `psake` tasks. They just don't all work OOB, I'll dig in more.
